changelogs/unreleased/drop-deprecated-log_msg-method.yml

```diff
@@ -0,0 +1,5 @@
+description: Drop deprecated log_msg method in the handler.
+change-type: patch
+destination-branches: [master]
+sections:
+  deprecation-note: "{{description}}"
```

Comment:
On the fence about including a deprecation-note or not.

Reply:
Yes, definitely. Keep in mind this will only be released (disregarding OSS) in about half a year, so the deprecation warning will have been there for that amount of time, therefor this is an end-user visible change.